### PR TITLE
Feature/module2

### PR DESCRIPTION
**Pull request by aidilumarov**

### CODE REVIEW

File: src/CatalogService/CatalogService.API/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["TicketApp.API/TicketApp.API.csproj", "TicketApp.API/"]
```
Comment (krylatij) [inline src/CatalogService/CatalogService.API/Dockerfile]: what is TicketApp.API?

File: src/CatalogService/CatalogService.Application/CategoryService.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
+﻿using CatalogService.Domain.Entities;
+using CatalogService.Persistence.Repositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace CatalogService.Application
+{
+    public class CategoryService
+    {
+        private readonly IRepository<Category> _categoryRepository;
```
Comment (krylatij) [inline src/CatalogService/CatalogService.Application/CategoryService.cs]: Services are more about onion architecture which is quite similar to  the clean architecture. But clean architecture should have "use-cases" which are more atomic than services (services incapsulates many "use-cases"). This video can help you to see the difference between these 2 approaches https://www.youtube.com/watch?v=KqWNtCpjUi8&ab_channel=MilanJovanovi%C4%87

**Review by krylatij — CHANGES_REQUESTED** [review]

File: src/CatalogService/CatalogService.API/Dockerfile (hunk 1 above)
Comment (aidilumarov) [reply src/CatalogService/CatalogService.API/Dockerfile]: I have reused one of my older projects and forgot to change this place. I will fix in the next commit

File: src/CatalogService/CatalogService.Application/CategoryService.cs (hunk 2 above)
Comment (aidilumarov) [reply src/CatalogService/CatalogService.Application/CategoryService.cs]: I have watched the video and converted the project to Clean Architecture. Thanks!

**Review by krylatij — APPROVED** [review]

**Pull request merged**